docs/modeling/compound-models.rst

```diff
@@ -176,6 +176,12 @@ In a future version it may be possible to "freeze" a compound model, so that
 from the user's perspective it is treated as a single model.  However, as this
 is the default behavior it is good to be aware of.
 
+One is also able to get the number of components (also known as submodels) in
+a compound model by using ``get_n_submodels`` method::
```

Comment:
The mention of "method" and its name needs to be corrected to reflect the correct new property.